README.md

```diff
@@ -1,2 +1,72 @@
-# flock
-Twitter on Dgraph
+## Flock 
```

Comment:
The headings do not look right. This is heading 2 whereas I see heading 1 below.